app/src/main/java/org/fossasia/phimpme/trashbin/TrashBinActivity.java

```diff
@@ -62,6 +76,20 @@
         setUpUi();
     }
 
+    private void trashEmptyViewSetup(){
+        if(ThemeHelper.getBaseTheme(getApplicationContext()) == ThemeHelper.DARK_THEME ||
+                ThemeHelper.getBaseTheme(getApplicationContext()) == ThemeHelper.AMOLED_THEME){
+            emptyIcon.setImageResource(R.drawable.ic_delete_sweep_white_24dp);
+            trashText.setTextColor(getTextColor());
+            trashMessage.setTextColor(getTextColor());
+
+        } else {
+            emptyIcon.setImageResource(R.drawable.ic_delete_sweep_black_24dp);
+            trashText.setTextColor(getTextColor());
+            trashMessage.setTextColor(getTextColor());
```

Comment:
I am sorry that I didn't mention this last time, but since these two lines are common for both if block and the else block, it can be moved outside the blocks.